SECURITY.md

```diff
@@ -0,0 +1,71 @@
+# Security Release Process
+
+The community has adopted this security disclosure and response policy to ensure we responsibly handle critical issues.
+
+
+## Supported Versions
+
+For a list of support versions that this project will potentially create security fixes for, please refer to the Releases page on this project's GitHub and/or project related documentation on release cadence and support.
+
+
+## Reporting a Vulnerability - Private Disclosure Process
+
+Security is of the highest importance and all security vulnerabilities or suspected security vulnerabilities should be reported to this project privately, to minimize attacks against current users  before they are fixed. Vulnerabilities will be investigated and patched on the next patch (or minor) release as soon as possible. This information could be kept entirely internal to the project.
+
+If you know of a publicly disclosed security vulnerability for this project, please **IMMEDIATELY** contact the maintainers of this project privately. The use of encrypted email is encouraged.
```

Comment:
We mention `the maintainers` of this project, but we don't have a `MAINTAINERS.rst` file as for example `python-tuf`: https://github.com/theupdateframework/python-tuf/blob/develop/docs/MAINTAINERS.txt
Maybe it's good to add one? @rdimitrov and @kairoaraujo

Reply:
+1